### ISSUE TITLE
Android application crashes at CachedImageRenderer.Init(true);

### ISSUE THREAD

**Issue by gaurangnirmal** [luberda-molinet/FFImageLoading#1260]
### Description
Purpose - SVG Image rendering.
Getting error something like this at 'CachedImageRenderer.Init(true)' initialization in Andriod project,
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: VTable setup of type FFImageLoading.Forms.Platform.CachedImageFastRenderer failed

### Actual Behavior
```
Unhandled Exception:

System.TypeLoadException: VTable setup of type FFImageLoading.Forms.Platform.CachedImageFastRenderer failed

05-03 14:49:22.996 E/mono    ( 7203): referenceTable GDEF length=814 1
referenceTable GSUB length=11364 1
referenceTable GPOS length=47302 1

05-03 14:49:22.996 E/mono    ( 7203): Unhandled Exception:
05-03 14:49:22.996 E/mono    ( 7203): System.TypeLoadException: VTable setup of type FFImageLoading.Forms.Platform.CachedImageFastRenderer failed
05-03 14:49:22.996 E/mono    ( 7203):   at (wrapper dynamic-method) System.Object.7(intptr,intptr,intptr)
05-03 14:49:22.996 E/mono    ( 7203):   at (wrapper native-to-managed) System.Object.7(intptr,intptr,intptr)
05-03 14:49:22.996 E/mono-rt ( 7203): [ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: VTable setup of type FFImageLoading.Forms.Platform.CachedImageFastRenderer failed
05-03 14:49:22.996 E/mono-rt ( 7203):   at (wrapper dynamic-method) System.Object.7(intptr,intptr,intptr)
05-03 14:49:22.996 E/mono-rt ( 7203):   at (wrapper native-to-managed) System.Object.7(intptr,intptr,intptr)
```

### Basic Information
Environment details-
CachedImage library version - 2.4.6.929
Xamarin.Forms version - 3.6.0.344457
I am using Visual Studio 2017 having version - 15.9.5

### Screenshots

![2019-05-03 14_53_34-](https://user-images.githubusercontent.com/12062404/57129214-5a1e0080-6db3-11e9-96c4-cbb8bbd0fb0a.png)

**Comment by daniel-luberda** [comment]
Please make sure you:
- have included **all** necessary packages in **all** projects
- all packages all in the same version
- delete all `bin`/`obj` folders from all projects & rebuild

**Comment by gaurangnirmal** [comment]
Nope, Man. still stuck with the same issue. Tried obj/bin folders. Restarted my IDE and tried to deployed. but still facing the same exception.

I even checked in my android project, It is also having the package: Xamarin.FFImageLoading.Svg.Forms 
 with version 2.4.6.929. Kindly, let me know if i am missing anything from this for SVG Image.

**Comment by daniel-luberda** [comment]
https://github.com/luberda-molinet/FFImageLoading/wiki/Xamarin.Forms-API#step-1

**Comment by TingtingAn** [comment]
@gaurangnirmal   did you get the solution?  I got same error.

**Comment by Anil86** [comment]
I\`m also getting the same error.  
When I use `Xamarin.Forms` built-in `Image`, app doesn\`t crash.

**Comment by gri4** [comment]
Just FYI, had the same issue. In my case i had to upgrade all xamarin.forms nugets to latest stable version (plus i did what is mentioned above). Issue had gone.
